server/engine/src/main/java/org/eclipse/lsp/cobol/service/delegates/completions/CompletionStorage.java

```diff
@@ -48,17 +49,18 @@ public abstract class CompletionStorage {
    * settings
    */
   public void updateStorage() {
-    settingsService.getConfiguration(DIALECTS.label).thenAccept(this::getDialects);
+    this.settingsService.getConfiguration(DIALECTS.label).thenAccept(this::setDialects);
   }
 
-  protected abstract InputStream getInputStream(List<String> dialectList);
+  protected abstract Map<String, T> getInputStream(List<String> dialectList);
```

Comment:
This name does not reflect the meaning of this method. It returns a map, not a stream
Please change it